src/aqua/decider/mailbox.aqua

```diff
@@ -0,0 +1,54 @@
+module Mailbox declares poll_mailbox, push_remove_worker
+
+export poll_mailbox, push_remove_worker
+
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+import PeerId from "@fluencelabs/aqua-lib/builtin.aqua"
+import Worker from "@fluencelabs/aqua-lib/workers.aqua"
+
+import Console from "../debug.aqua"
+import SpellId, WorkerId from "../types.aqua"
+
+import spell_log from "../fluence/spell.aqua"
+
+data RemoveWorker:
+    host_id: PeerId
+    worker_id: WorkerId
+
+data DeciderMessage:
+    remove: ?RemoveWorker
+
+service JsonMsg("json"):
+  parse(str: string) -> DeciderMessage
+  stringify(msg: DeciderMessage) -> string
+
+func remove_worker(worker: RemoveWorker):
```

Comment:
I suggest to place remove_worker near Worker.create from ../fluence/workers.aqua

Reply:
I'll remove the whole `mailbox.aqua`, it doesn't work properly imo